packages/manager/src/features/Volumes/VolumesLanding.tsx

```diff
@@ -166,6 +175,14 @@ export const VolumesLanding = () => {
   return (
     <>
       <DocumentTitleSegment segment="Volumes" />
+      {_isRestrictedUser && (
+        <Notice variant="warning" important>
```

Comment:
```suggestion
        <Notice variant="error" important>
```
Looking at some of the other restricted user messages, we usually use error variants. I'd recommend we stay consistent here, unless there was a reason we decided to go with warning.

Reply:
@mjac0bs The epic has more details on this: [https://jira.linode.com/browse/M3-7680](https://jira.linode.com/browse/M3-7680)
But is this page considered a "create" page or a "detail" page?

Reply:
😅 ¿Por que no los dos? With Volumes, this is all we've got.

Reply:
I'm okay with warning, the more I think about it, but I don't think there's a right answer here unless UX has specific guidance.